src/ontology/mondo-edit.obo

```diff
@@ -500399,8 +500392,6 @@ xref: OMIM:619478 {source="MONDO:equivalentTo"}
 xref: UMLS:C5561960 {source="MEDGEN:1794170", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0001347 {source="DOID:0060918", source="OMIM:619478"} ! facioscapulohumeral muscular dystrophy
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2979 {source="OMIM:619478"} ! DNMT3B
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 {source="OMIM:619478"} ! DUX4
```

Comment:
This term has the annotation
`excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql`

These gene annotations should not be removed.

Reply:
So the presence of the `http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql` _means_ necessarily, that this is a disease which is allowed to have multiple genes as causes - this will make our pipelines a little more complex..

Reply:
yes, the `http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql` means a curator reviewed this and agreed that this disease has more than 1 disease defining gene associations